src/protocols/secure_channel/PairingSession.h

```diff
@@ -174,6 +174,17 @@ class DLL_EXPORT PairingSession : public SessionDelegate
         else
         {
             err = OnFailureStatusReport(report.GetGeneralCode(), report.GetProtocolCode());
+
+            if (report.GetGeneralCode() == Protocols::SecureChannel::GeneralStatusCode::kBusy &&
+                report.GetProtocolCode() == Protocols::SecureChannel::kProtocolCodeBusy)
+            {
+                uint16_t minWaitTime = 0;
+                if (!report.GetProtocolData().IsNull() && report.GetProtocolData()->DataLength() >= sizeof(minWaitTime))
+                {
+                    memcpy(&minWaitTime, report.GetProtocolData()->Start(), sizeof(minWaitTime));
```

Comment:
This is not doing the right thing with endianness.  Please use a LittleEndian::Reader to read the value.

Please file a followup issue to propagate this value along with our error notification, so consumers actually know when to retry....

Reply:
Using the `LittleEndian::Reader` and filed #28290